lib/plain/common-types.ts

```diff
@@ -672,6 +675,9 @@ export type PlainClientAPI = {
     getMany(
       params: OptionalDefaults<GetSpaceEnvironmentParams & PaginationQueryParams>
     ): Promise<CollectionProp<AppInstallationProps>>
+    getForOrganization(
+      params: OptionalDefaults<GetAppDefinitionParams>
```

Comment:
Shouldn't we have a type with a different name here? Meaning it mentions App Definition, but lives in App Installation.

Reply:
We can make but I think it will be same type like `GetAppDefinitionParams` just with  different name so I just reused it. 
@andipaetzold wdyt?

Reply:
No strong opinion here.
I think it is fine to keep things like they are as I doubt many people read these types. Also in the existing code, we often reuse types. But feel free to create an alias type with an easier to remember name.

Reply:
Alright, I have no strong knowledge of this codebase, so was just asking.